helpers/tipos/funcionarios.go

```diff
@@ -0,0 +1,128 @@
+package tipos
+
+import (
+	"fmt"
+
+	"github.com/astaxie/beego"
+	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/terceros_mid/models"
+	"github.com/udistrital/utils_oas/request"
+)
+
+// GetAllFuncionariosPlanta trae los funcionarios de planta
+func GetAllFuncionarios(idTercero int) (terceros []map[string]interface{}, outputError map[string]interface{}) {
```

Comment:
- Ajustar/Completar descripción y...
- nombre de esta función (se puede omitir el `All` teniendo en cuenta el propósito de estos helpers... inclusive se podría omitir el `Get`, pero bueno, se podría dejar por consistencia...)
```suggestion
// GetFuncionarios trae ...?
func GetFuncionarios(idTercero int) (terceros []map[string]interface{}, outputError map[string]interface{}) {
```